.github/workflows/stale-bot.yml

```diff
@@ -0,0 +1,25 @@
+# https://github.com/actions/stale
+
+name: Close Inactive Issues and Pull Requests
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  close-issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 90
+          days-before-issue-close: 14
```

Comment:
I would not close issues automatically. I would review the stale issues in the backlog meeting and close them there or update them to get another 90 days.